yazi-core/src/input/commands/kill.rs

```diff
@@ -4,13 +4,23 @@ use yazi_shared::{event::Exec, CharKind};
 
 use crate::input::Input;
 
+#[derive(Debug, Copy, Clone)]
+enum KillMode {
+	Default,
+	BigWord,
+}
+
 pub struct Opt<'a> {
 	kind: &'a str,
+	mode: KillMode,
 }
 
 impl<'a> From<&'a Exec> for Opt<'a> {
 	fn from(e: &'a Exec) -> Self {
-		Self { kind: e.args.first().map(|s| s.as_str()).unwrap_or_default() }
+		Self {
+			kind: e.args.first().map(|s| s.as_str()).unwrap_or_default(),
+			mode: e.named.get("big-word").map_or(KillMode::Default, |_| KillMode::BigWord),
```

Comment:
Is better to impl the `FromStr` (also `Default`) trait for the `KillMode` to do this.